gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -439,6 +439,42 @@ def create_revel_grch38_ht() -> hl.Table:
     return final_ht
 
 
+def create_phylop_grch38_ht() -> hl.Table:
+    """
+    Convert PhyloP scores to Hail Table.
+
+    .. note::
+    BigWig format of Phylop was download from here:
+    https://cgl.gi.ucsc.edu/data/cactus/241-mammalian-2020v2-hub/Homo_sapiens/241-mammalian-2020v2.bigWig
+    and converted it to bedGraph format with bigWigToBedGraph from the kent packages
+    of UCSC (https://hgdownload.cse.ucsc.edu/admin/exe/) with the following command:
+    `./bigWigToBedGraph ~/Downloads/241-mammalian-2020v2.bigWig ~/Downloads/241-mammalian-2020v2.bedGraph`
+    The bedGraph file is bigzipped before importing to Hail.
+    Different to other in silico predictors, the Phylop HT is keyed by locus only. Since
+     the PhyloP scores have one value per position, we exploded the interval to store
+     the HT by locus.
+
+    :return: Hail Table with Phylop Scores for GRCh38
+    """
+    bg_path = "gs://gnomad-insilico/phylop/Human-GRCh38-Phylop-241-mammalian-2020v2.bedGraph.bgz"
+    columns = ["chr", "start", "end", "phylop"]
+    ht = hl.import_table(
+        bg_path,
+        min_partitions=1000,
+        impute=True,
+        no_header=True,
+    ).rename({f"f{i}": c for i, c in enumerate(columns)})
+
+    # We add 1 to both start and end because input bedGraph is 0-indexed interval and
+    # hl.range is end exclusive
```

Comment:
```suggestion
    # We add 1 to both start and end because input bedGraph is 0-indexed interval and
    # the interval is end exclusive
```
Sorry I meant the interval